platformcomponents/desktop/attachment-file.json

```diff
@@ -0,0 +1,28 @@
+{
+  "attachment": {
+    "file": {
+      "comment": "Applied to file attachments",
+      "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=5415%3A0",
+      "border": {
+        "#normal": {
+          "background": "@theme-common-outline-secondary-normal"
+        }
+      },
+      "icon": {
```

Comment:
is this for the bricklet icon?

Reply:
Yes it is for the bricklet icon.

Reply:
Good, then we should specify it.
`"attachmentfile":{
      "sent":{
          "bricklet":{
              "background": "@theme-...",
              "icon": "@theme-..."
           }
      }
}`

Reply:
Pushed commit _Updated used keys_ with the requested changes.
Changing this conversation to Resolved.